### DISCUSSION TITLE
Pacman mode

### DISCUSSION THREAD

**Discussion by Nettogrof** [BattlesnakeOfficial/feedback#25] [General]
TLDR: A mode with a map like a pacman game, with wall inside the battlefield.

This game mode will be on a large maps (maybe even larger than 19x19) with a maps with wall like in Pacman . That gonna need a new fields in payload  named like "wall".  

Movement and rules will be the same as standard mode. 
Could "reuse" the hazard field as "solid wall"   like hazard damage more the 100 health.
Food and Health as standard mode 

To see a example (single snake mode ) : https://archive.org/details/arcade_nibbler

**Comment by andrewdmaclean** [comment]
Love this idea!!!!
